packages/twenty-chrome-extension/codegen.ts

```diff
@@ -1,9 +1,13 @@
 import { CodegenConfig } from '@graphql-codegen/cli';
 
 const config: CodegenConfig = {
-  schema: ['http://localhost:3000/graphql'],
-  overwrite: true,
-  documents: ['./src/**/*.ts', '!src/generated/**/*.*'],
+  schema: [`${import.meta.env.VITE_SERVER_BASE_URL}/graphql`],
+  overwrite: false,
+  documents: ['./src/**/*.ts', 
+    '!src/generated/**/*.*', 
+    '!src/graphql/company/*', 
```

Comment:
@AdityaPimpalkar Why are we ignoring these folders? Are these queries/mutations not useful?

Reply:
Yes, seemed like a viable solution as those queries and mutations require authorization and they were standard objects so they won't change much any time soon, but I've changed and kept is as it was and just added authorization headers